src/types/state.ts

```diff
@@ -55,8 +55,8 @@ export interface TargetState {
 		apps: {
 			[appId: string]: {
 				name: string;
-				commit: string;
-				releaseId: number;
+				commit?: string;
```

Comment:
Are these fields really optional for v2?

Reply:
Yes, apps start out without any release at which point this will be null